### PR TITLE
PR For Issue 243 Allow Specification of Channel MetaData during Channel Creation

### PR DESCRIPTION
**Pull request by sgd2z**
https://github.com/finos/FDC3/issues/243

### CODE REVIEW

File: src/api/DesktopAgent.ts [hunk 1]
```diff
@@ -144,9 +144,11 @@ interface DesktopAgent {
      * It is up to applications to manage how to share knowledge of these custom channels across windows and to manage
      * channel ownership and lifecycle. 
      * 
+     * If `displayMetaData` is specified, the channel is visually selectable by the user. If the channel already exists, displayMetaData is ignored.
+     *
```
Comment (rikoe) [inline src/api/DesktopAgent.ts:148]: displayMetadata (not "MetaData")

File: src/api/DesktopAgent.ts [hunk 2]
```diff
@@ -144,9 +144,11 @@ interface DesktopAgent {
      * It is up to applications to manage how to share knowledge of these custom channels across windows and to manage
      * channel ownership and lifecycle. 
      * 
+     * If `displayMetaData` is specified, the channel is visually selectable by the user. If the channel already exists, displayMetaData is ignored.
+     * 
      * `Error` with a string from the `ChannelError` enumeration.
      */
-    getOrCreateChannel(channelId: string): Promise<Channel>;
+    getOrCreateChannel(channelId: string, displayMetaData?: DisplayMetaData): Promise<Channel>;
```
Comment (rikoe) [inline src/api/DesktopAgent.ts:151]: `displayMetadata?: DisplayMetadata`

**Review by rikoe — CHANGES_REQUESTED** [review]

**Comment by rikoe** [comment]
@sgd2z I propose we close this for the time being, as we haven't reached consensus for this one and it isn't currently planned for 1.2 (if I am correct), and this particular approach was observed as somewhat ambiguous/confusing by the standards working group.

**Comment by sgd2z** [comment]
> @sgd2z I propose we close this for the time being, as we haven't reached consensus for this one and it isn't currently planned for 1.2 (if I am correct), and this particular approach was observed as somewhat ambiguous/confusing by the standards working group.

I have a poll in the issue itself on which of two options to pick. I'll update the PR accordingly once a decision is made.

**Comment by nkolba** [comment]
@sgd2z  - I've commented on the related issue.  We haven't gotten response on the poll.  Should we close issue and PR for now and take up again when we have more supporting use cases in this area?

**Comment by kriswest** [comment]
Closing as the associated issue was closed

**Pull request closed without merging**